doc/cephfs/fs-nfs-exports.rst

```diff
@@ -216,6 +216,17 @@ where:
 
 ``binding`` is the pseudo root path (must be an absolute path).
 
+
+Update CephFS Export
+====================
+
+.. code:: bash
+
+    $ ceph nfs export update -i <json_file>
+
+This updates the cephfs export specified in the json file. Export in json
+format can be fetched with above get command.
```

Comment:
Is the json format documented somewhere?

Reply:
No

Reply:
At least give an example then.

Reply:
As well as an example of what `export get` returns.

Reply:
I have added example